### DISCUSSION TITLE
Switch to Auto Launch

### DISCUSSION THREAD

**Discussion by tonyoquinn** [X2CommunityCore/xcom2-launcher#321] [Ideas]
I'd like to be able to launch the game without stopping at the aml gui.  After I've played several times, there's no need check everything every time.  I recommend command line switches to auto launch xcom 2, like -LaunchLastPlayed or -LaunchXCOM2 etc.

**Comment by RevZero** [comment]
Thank you for the suggestion. Once you have your mods set up, you could essentially just run XCOM without starting AML at all. But I see how this could be useful. AML would obviously have to skip the initial mod processing. However, I take this under consideration.
